src/modules/job-info/watch.c

```diff
@@ -246,13 +246,24 @@ static void watch_continuation (flux_future_t *f, void *arg)
     const char *input;
     const char *tok;
     size_t toklen;
+    const char *errmsg = NULL;
```

Comment:
nit: was the commit message supposed to have "Fixes #4612" not just "Issue #4612"?

Reply:
oops, yeah